#!/usr/bin/python3
def raise_exception():
    a = 1 + '1'
    print("{:s}".format(a))
